packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchSuggestions.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+    <div :class="$style.shell">
+        <button
+            v-for="(item, index) in getAddressItems"
+            :key="index"
```

Comment:
Best to give the key a value which is more likely to be unique i.e. ``:key="`${index}_ getAddressItem`"``

Reply:
Yup, fixed.